sign/eddilithium3.go

```diff
@@ -0,0 +1,108 @@
+package sign
+
+import (
+	"github.com/cloudflare/circl/sign/eddilithium3"
+
+	cryptoRand "crypto/rand"
+	"encoding/asn1"
+	"errors"
+)
+
+type edDilithium3Scheme struct{}
```

Comment:
All these definitions must live inside of `sign/eddilithium3` package,

Reply:
That is what I would like as well, but how do you do that without creating an import cycle?